feature-b/FeatureB.swift

```diff
@@ -0,0 +1 @@
+// This is a placeholder for the feature B
```

Comment:
make feature capitalized
```suggestion
// This is a placeholder for the Feature B
```